content/tutorials/grafana-fundamentals.md

```diff
@@ -345,6 +345,15 @@ Once you've acknowledged an alert, consider pausing it. This can be useful to av
 1. Find your alert in the list, and click the **Pause** icon on the right. The **Pause** icon turns into a **Play** icon.
 1. Click the **Play** icon to resume evaluation of your alert.
 
+### Clean up the local environment
+
+The tutorial has running Docker containers.
```

Comment:
Please remove whatever is auto-ending your lines. It's a real pain for reviewing.